pkgs/top-level/all-packages.nix

```diff
@@ -577,6 +577,8 @@ in
 
   argtable = callPackage ../tools/misc/argtable {};
```

Comment:
I think it would be nice to rename the existing `argtable` to `argtable3` (both attribute and directory names).
And also define
```
argtable = argtable3;
```

It seems argtable3 (release 2017) is well maintained then argtable2 (release 2011) so we should encourage users to use argtable3. What do you think?

Reply:
:+1: is it possible to change the `argtable` directory name - or are there other implications ?

Reply:
Only the attribute name is used by other derivations. So, you can change the directory path.